src/project/qml/MuseScore/Project/internal/Export/ExportOptionsView.qml

```diff
@@ -122,21 +126,25 @@ Column {
         id: pageLoader
         width: parent.width
 
+        function refresh() {
+            if (!root.exportModel.selectedExportType.settingsPagePath) {
+                setSource("")
```

Comment:
@cbjeukendrup

Should I prefix this with `pageLoader` for readability, or is the object `setSource()` acts on clear enough contextually?

```suggestion
                pageLoader.setSource("")
```

(also at line 140)

Reply:
It's fine like this! Thanks!